packages/components/src/docs/General/Accessibility.mdx

```diff
@@ -0,0 +1,74 @@
+import { Meta } from '@storybook/blocks';
```

Comment:
the file needs to be renamed into "7-Accessibility" right @mariohamann?

I would even rename it to "6-Accessibility" and rename the browser issue to "7-Known Browser Issues"